packages/components/organisms/f-footer/package.json

```
@@ -40,11 +40,11 @@
   ],
   "dependencies": {
     "@justeat/f-services": "1.3.0",
-    "@justeat/f-vue-icons": "1.10.0",
     "v-click-outside": "3.1.2"
   },
   "peerDependencies": {
-    "@justeat/f-trak": ">=0.6.0"
+    "@justeat/f-trak": ">=0.6.0",
+    "@justeat/f-vue-icons": "1.10.0"
```

We don't want to pin a single version as a peer dependency, you can set a range here instead.

```suggestion
    "@justeat/f-vue-icons": ">=1.10.0"
```